pkgs/development/embedded/avrdude/default.nix

```diff
@@ -48,11 +48,6 @@ stdenv.mkDerivation (finalAttrs: {
   cmakeFlags = lib.optionals docSupport [ "-DBUILD_DOC=ON" ]
     ++ lib.optionals stdenv.hostPlatform.isLinux [ "-DHAVE_LINUXSPI=ON" "-DHAVE_PARPORT=ON" ];
 
-  # dvips output references texlive in comments, resulting in a huge closure
-  postInstall = lib.optionalString docSupport ''
-    rm $out/share/doc/avrdude/*.ps
-  '';
```

Comment:
Is this no longer the case?

Reply:
No. The current version does not generate .ps files during the build phase which leads to failing this phase. As the files which should be deleted do not exist in the first place this phase should be able to be safely removed.